src/core_plugins/tests_bundle/webpackShims/angular-mocks.js

```diff
@@ -0,0 +1,2 @@
+require('angular');
+require('../../../../node_modules/angular-mocks/angular-mocks.js');
```

Comment:
I'm afraid I'm missing how `require('angular')` works and `require('angular-mocks')` doesn't, not doubting you, just curious.